index.js

```diff
@@ -16,43 +52,57 @@ module.exports = {
       name: options.name,
 
       defaultConfig: {
-        configuredServices: {
-          bugsnag: {
-            url: 'http://notify.bugsnag.com/deploy'
+        configuredServices: function(context) {
+          return {
+            bugsnag: {
+              url: 'http://notify.bugsnag.com/deploy',
+              method: 'POST',
+              headers: {},
+              body: function() {
+                var apiKey = this.apiKey;
+
+                if (!apiKey) { return; }
```

Comment:
Ah, I see. But shouldn't users have to opt-in into services anyway? Wouldn't the current implementation mean that the Bugsnag service would always be used as soon as the addon is installed? I'd say users should have to explicitly opt-in by at least configuring

```js
bugsnag: {
  apiKey: 'some',
  didActivate: true
}
```

and with only

```js
bugsnag: {
  apiKey: 'some'
}
```

it would just not do anything. Maybe we should have a warning for that case then saying that bugsnag was configured but not actually activated for any hook. I think that way it would be more explicit.